operations/customers.md

```diff
@@ -242,6 +248,7 @@ Searches for customers that are active at the moment in the enterprise \(e.g. co
 | `Client` | string | required | Name and version of the client application. |
 | `Name` | string | optional | Name to search by \(applies to first name, last name and full name\). |
 | `ResourceId` | string | optional | Identifier of [Resource](enterprises.md#resource) to search by \(members of [Reservation](reservations.md#reservation) assigned there will be returned\). |
+| `Extent` | [Customer extent](customers.md#customer-extent) | required | Extent of data to be returned. |
```

Comment:
```suggestion
| `Extent` | [Customer extent](customers.md#customer-extent) | optional | Extent of data to be returned. |
```
I don't think it's required

Reply:
@vitezp Optional is the old way. We always state extents are required. this forces people to think about the solution.

Reply:
Good, @stang704 consider this irrelevant.

Reply:
@vitasamek would you prefer then if we put it as `required` in the documentation even though it's not strictly enforced by our system?

Reply:
> We always state extents are required. Of course, it has to be done in a non-breaking way.(optional in the code)

That's what I wrote. In the documentation, it's defined as required. The code doesn't verify it.

That means, that after some time, we can put the request check and we can say: It's defined as requested all the time, you didn't follow documentation.

If we would do it vice versa, we would have to notify all of them and wait for the deprecation period.

Reply:
Extent returned back to `required` :D